cmd/query/app/grpc_handler.go

```diff
@@ -85,7 +85,7 @@ func NewGRPCHandler(queryService *querysvc.QueryService,
 		queryService:        queryService,
 		metricsQueryService: metricsQueryService,
 		logger:              options.Logger,
-		tracer:              options.Tracer,
+		tracer:              *options.Tracer,
```

Comment:
there should be no pointer dereferencing, please make sure the type is always `*JTracer`